x-pack/plugins/index_management/public/application/sections/home/index_list/details_page/details_page_overview/status_details.tsx

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent, ReactNode } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiBadge, EuiFlexGroup, EuiFlexItem, EuiIcon, EuiText, EuiTextColor } from '@elastic/eui';
+import { css } from '@emotion/react';
+import { euiThemeVars } from '@kbn/ui-theme';
+
+import { useAppContext } from '../../../../../app_context';
+import { Index } from '../../../../../../../common';
+import { OverviewCard } from './overview_card';
+
+export const StatusDetails: FunctionComponent<{
+  documents: Index['documents'];
+  documentsDeleted: Index['documents_deleted'];
+  status: Index['status'];
+  health: Index['health'];
+}> = ({ documents, documentsDeleted, status, health }) => {
+  const { config } = useAppContext();
+  if (!config.enableIndexStats) {
+    return null;
+  }
+  let healthBadge: ReactNode;
+  if (health === 'green' || health === 'GREEN') {
```

Comment:
would this be a scenario we might run into (discrepancies in capitalization)? Otherwise we could try to normalize it like: `health.toLowerCase() === 'green'`.. etc

Bit nit picky, but we could also avoid having to assign Badges to a variable if we use an object lookup to define the badge:

```typescript

const healthBadge = {
  red: {
    color: 'danger',
    label: 'Critical'
  },
  ...
};

const healthStatus = healthBadge[health];
const healthBadge = (
  <EuiBadge color={healthStatus.color}>
      {healthStatus.label}
  </EuiBadge>
);

```

Reply:
That's a great suggestion, @sabarasaba, thanks! I'll add it to the next commit.